tests/trackers/datafixtures/eniahd.html

```diff
@@ -0,0 +1,1234 @@
+<!DOCTYPE html>
```

Comment:
Let's keep datafixtures minimal. 
Please cut out unimportant stuff from html (see nnm fixture for example).

Reply:
I've removed all non-related stuff from the HTML.
401 insertions(+), 1224 deletions(-)

hope it helps